datafusion/sqllogictest/test_files/string_view.slt

```diff
@@ -270,37 +270,37 @@ explain SELECT column1_utf8 from test where 'Andrew' = column1_utf8view;
 ----
 logical_plan
 01)Projection: test.column1_utf8
-02)--Filter: test.column1_utf8view = Utf8View("Andrew")
+02)--Filter: CAST(Utf8("Andrew") AS Utf8View) = test.column1_utf8view
 03)----TableScan: test projection=[column1_utf8, column1_utf8view]
 
 query TT
 explain SELECT column1_utf8 from test where column1_utf8 = arrow_cast('Andrew', 'Utf8View');
 ----
 logical_plan
-01)Filter: test.column1_utf8 = Utf8("Andrew")
+01)Filter: CAST(test.column1_utf8 AS Utf8View) = CAST(Utf8("Andrew") AS Utf8View)
```

Comment:
I think this is not a good change -- it shows the column itself being cast (which is a substantial overhead at runtime)

Reply:
Totally agree! I'm aiming at restoring most of the sqllogictests (and this behaviour should be easy to catch) to their original expected results so consider the edits just as a way to make the test progress so it can surface more weird behaviours.